views/detail.ejs

```diff
@@ -1,2 +1,9 @@
 <% // Render individual movie details here %>
 <% // This is how to do comments in EJS, by the way... %>
+<h1><%= data.Title  %> /h1>
+    <img src="<%=data.Poster%> " alt="<%=data.title %> Poster ">
+    <p><%= data.Plot %> </p>
+    <p><%= data.Director %> </p>
+    <p><%= data.Writer %> </p>
+    <p><%= data.Actors %> </p>
+    <p><%= data.Awards %> </p>
```

Comment:
your file is called detail.ejs **singular** but you are trying to render a 'details' **plural** page on line 63 in your server.js file, so one of those needs to change.